rclpy/rclpy/node.py

```diff
@@ -92,16 +92,21 @@
 
 
 class Node:
-
-    PARAM_REL_TOL = 1e-6
-
     """
     A Node in the ROS graph.
 
     A Node is the primary entrypoint in a ROS system for communication.
     It can be used to create ROS entities such as publishers, subscribers, services, etc.
     """
 
+    PARAM_REL_TOL = 1e-6
+    """
+    Maximum differnance between two floating point values to determine that they are 'close'.
```

Comment:
@dawonn-haval consider: 
```suggestion
    Relative tolerance for floating point parameter values' comparison. See `math.isclose` documentation.
```
to avoid repeating documentation.